packages/app/src/Feed/TimelineFeed.ts

```diff
@@ -49,8 +50,11 @@ export default function useTimelineFeed(subject: TimelineSubject, options: Timel
         sub.PTags = new Set(subject.items);
         break;
       }
-      case "keyword": {
-        sub.Kinds.add(EventKind.SetMetadata);
+      case "profile_keyword": {
+        sub.Search = subject.items[0];
```

Comment:
You can set the kind in here instead then you dont need that if statement above

Reply:
All subject.types except profile_keyword have TextNote & Repost kind, profile_keyword only has SetMetadata only. Seems like that if is necessary, or some version of it?

Reply:
Yea just overwrite the kinds in here and leave the old default kinds above without if statement?

```
sub.Kinds = new Set([a,b]);
...
case "profile_keyword": 
  sub.Kinds = new Set[c]);
  ...
```

Reply:
Ok, that's simpler, done.